### DISCUSSION TITLE
Notifications won’t refresh on mobile

### DISCUSSION THREAD

**Discussion by jahabeebs** [community/community#78076] [Mobile]
When trying to refresh my notifications on mobile the request simply hangs and a loading spinner is shown. The request never completes. Clearing the cache and restarting the app does not work.

![image](https://github.com/community/community/assets/47253537/972280ad-ba19-4dbf-8d90-b3fa6600bd91)

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.141.0 (116980603)
Device: iPhone15,2 (iOS 17.1.2)
</details>

**Comment by peterroth** [comment]
Facing the exact same behavior on Android 10.
